src/views/SystemIntake/ViewOnly/index.tsx

```diff
@@ -7,11 +8,66 @@ import { SystemIntakeForm } from 'types/systemIntake';
 type SystemIntakeViewOnlyProps = {
   systemIntake: SystemIntakeForm;
 };
-const SystemIntakeView = ({ systemIntake }: SystemIntakeViewOnlyProps) => (
-  <>
-    <h1>Review your Intake Request</h1>
-    <SystemIntakeReview systemIntake={systemIntake} now={DateTime.local()} />
-  </>
-);
+
+function downloadFile(filename: string, content: string) {
+  axios
+    .request({
+      url: `${process.env.REACT_APP_API_ADDRESS}/pdf/generate`,
+      responseType: 'blob',
+      method: 'POST',
+      data: {
+        html: content,
+        filename
+      }
+    })
+    .then(response => {
+      // This approach works fine in the tests I've done in Chrome
+      // With PDF files that are < 100kB. For larger files we might need to
+      // instead redirect the browser to a URL that returns the file. That
+      // approach is complicated by using JWTs for auth.
+      const blobUrl = URL.createObjectURL(
+        new Blob([response.data], { type: 'application/pdf' })
+      );
+
+      const link = document.createElement('a');
+
+      link.href = blobUrl;
+
+      // TODO generate a useful filename here
+      link.download = 'generated.pdf';
+
+      // This downloads the file
+      // TODO test this in various browsers. Some reports say this might not work
+      // properly in Firefox and that firing a MouseEvent is required instead.
+      document.body.appendChild(link);
```

Comment:
If we stored the response in the state for this component (or in redux, even), couldn't we use standard JSX to show/hide the link and handle the click/redirect?

Reply:
I'd have to try it, but I think that might work.